lib/puppet/type/cisco_bgp_neighbor_af.rb

```diff
@@ -387,6 +389,24 @@ def self.title_patterns
     newvalues(:true, :false, :default)
   end
 
+  newproperty(:route_map_in) do
+    desc 'route-map in state. Valid values are a string defining the name ' \
```

Comment:
We used to provide more error checking... for instance, won't this error with a cryptic message if the user passes in an integer or fixnum instead of a string?